baton-maven-plugin/src/main/java/org/technologybrewery/baton/util/CommonUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.technologybrewery.baton.util;
+import com.vdurmont.semver4j.Semver;
+
+public class CommonUtils {
```

Comment:
S: all static classes like this should be `final` with a private constructor to prevent instantiation.  A tool like SonarLint can be useful  for stuff like this.